projects/plugins/beta/src/admin/notice.template.php

```diff
@@ -39,6 +39,19 @@
 				margin-bottom:1em;
 			}
 		</style>
+
+		<?php
+		if ( isset( $plugin ) ) {
+			$plugin = $plugin; // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+			?>
+			<div id="jetpack-beta-tester__breadcrumb">
+				<a href="<?php echo esc_url( Utils::admin_url() ); ?>">
+					<?php esc_html_e( 'Jetpack Beta Tester Home', 'jetpack-beta' ); ?>
+				</a>
+				<span>&nbsp;&gt; <?php echo esc_html( $plugin->get_name() ); ?></span>
+			</div>
+		<?php } ?>
```

Comment:
This is not the right place for this. Putting it here makes it not work on JN (or, more specifically, it stops working as soon as a dev version of a plugin has been installed). Plus it's not semantically the right place for it.

Try somewhere around [plugin-manage.template.php lines 75–76](https://github.com/Automattic/jetpack/blob/693568d7721b8d7fe201f7059c76a16141b1cb2b/projects/plugins/beta/src/admin/plugin-manage.template.php#L75-L76) instead. 

Or maybe work it into [the header](https://github.com/Automattic/jetpack/blob/693568d7721b8d7fe201f7059c76a16141b1cb2b/projects/plugins/beta/src/admin/header.template.php#L29) without adding a new line. BTW, that header can be clicked to get back, but that's not at all obvious.

Reply:
Ah `plugin-manage.template.php` does seem like the right place for this 👍   https://github.com/Automattic/jetpack/pull/20394/commits/6a84fb2ebeb8616021ebb11751caa4b2266d01a7